spec/requests/verifications_spec.rb

```diff
@@ -9,6 +9,67 @@
     create(:verification, audio_event:, creator: writer_user, confirmed: Verification::CONFIRMATION_FALSE)
   end
 
+  let(:second_event) { create(:audio_event, audio_recording:, creator: writer_user) }
+
+  it 'can update a verification' do
+    payload = {
+      verification: {
+        confirmed: Verification::CONFIRMATION_UNSURE
+      }
+    }
+    patch "/verifications/#{verification.id}",
+      params: payload, **api_with_body_headers(writer_token)
+
+    expect(response).to have_http_status(:ok)
+    expect(api_data).to include(
+      audio_event_id: audio_event.id,
+      creator_id: writer_user.id,
+      tag_id: tag.id,
+      confirmed: Verification::CONFIRMATION_UNSURE
+    )
+  end
+
+  it 'can upsert a verification when a matching record exists' do
+    payload = {
```

Comment:
mainly for readability, but can we assert the pervious state of the verification?